src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -290,6 +290,38 @@ private static void AddPopParams(AuthenticationRequestParameters authenticationR
 
             return msalTokenResponse;
         }
+        
+        /// <summary>
+        /// Token Received from WAM
+        /// </summary>
+        /// <param name="authResult"></param>
+        /// <param name="logger"></param>
+        private static bool TokenReceivedFromWam(
+            NativeInterop.AuthResult authResult,
+            ILoggerAdapter logger)
+        {
+            //success result from WAM
+            if (authResult.IsSuccess)
+                return true;
+            
+            //user switch result is not success and a token is received from MSALRuntime with an Error Response status    
+            if (authResult.Error != null 
+                && (ResponseStatus)authResult.Error.Status == ResponseStatus.UserSwitch)
+            {
+                string internalErrorCode = authResult.Error.Tag.ToString(CultureInfo.InvariantCulture);
+
+                string message =
+                        $" Error Code: {authResult.Error.ErrorCode} \n" +
+                        $" Error Message: {authResult.Error.Context} \n" +
+                        $" Internal Error Code: {internalErrorCode} \n";
```

Comment:
Since this is considered a success scenario, do we really need this log? Can we just log that account was switched?